cfme/fixtures/pods_per_ready_status.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+from wrapanapi.utils import eval_strings
+
+
+@pytest.fixture(scope='function')
+def pods_per_ready_status(provider):
```

Comment:
I dont see a need for this to be a fixture, this could just be a method on the container object